src/lib/core/NodeId.h

```diff
@@ -45,8 +45,9 @@ constexpr NodeId kPlaceholderNodeId   = 0xFFFF'FFFE'FFFF'FFFFULL;
 constexpr NodeId kMinCASEAuthTag = 0xFFFF'FFFD'0000'0000ULL;
 constexpr NodeId kMaxCASEAuthTag = 0xFFFF'FFFD'FFFF'FFFFULL;
 
-constexpr NodeId kMinPAKEKeyId = 0xFFFF'FFFB'0000'0000ULL;
-constexpr NodeId kMaxPAKEKeyId = 0xFFFF'FFFB'FFFF'FFFFULL;
+constexpr NodeId kMinPAKEKeyId  = 0xFFFF'FFFB'0000'0000ULL;
+constexpr NodeId kMaxPAKEKeyId  = 0xFFFF'FFFB'FFFF'FFFFULL;
+constexpr NodeId kMaskPAKEKeyId = 0x0000'0000'0000'FFFFULL;
```

Comment:
Just trying to confirm whether PAKE key ID range is supposed to be 16-bits (which is what I thought) or 32-bits (as it looks like it is according to min/max in range).

Reply:
PAKE key ID is same as PasscodeID which is defined as uint16 in spec:
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/ACL-Cluster.adoc#adminpasscodeid

AdminPasscodeID | uint16

In SDK, this field is also defined as unint16_t in class DLL_EXPORT PASESession
uint16_t mPasscodeID = 0;

But 5.1.1.6. Passcode has following defination.
A Passcode SHALL be included as a 27-bit unsigned integer, which serves as proof of possession
during commissioning. 

It seems there are some discrepancy in the spec, we need first fix the spec discrepancy